src/Core/ConfigurationParser.re

```diff
@@ -380,6 +380,13 @@ let configurationParsers: list(configurationTuple) = [
     "workbench.editor.showTabs",
     (config, json) => {...config, workbenchEditorShowTabs: parseBool(json)},
   ),
+  (
+    "workbench.sideBar.position",
```

Comment:
Small thing, but could this be swapped to `"workbench.sideBar.location"`? I think that `"position"` makes more sense really, its not a layout...but it would keep us in sync with the vscode config option I believe, which we attempt to do where it makes sense.

Reply:
Yeah, I'm happy to do whatever's best for the project. Not torn up about that. I'll make an update later to this. Do I need to squash commits when I do for this repo?

Reply:
Nah, we will do a squash commit at the end, so don't worry about that.

Reply:
Resolved.